submit.html

```diff
@@ -1,5 +1,63 @@
---- 
-layout: default 
+---
+layout: default
 title: Submit a talk
 description: How to become a speaker and how to submit a talk
----
\ No newline at end of file
+---
+
+<p>
+  Thank you for considering to take part in A11y Meetup Berlin, we're very excited to read your talk proposal!
+</p>
+
+<section>
+  <h2>Topics</h2>
+  <p>
+    We welcome everyone who would like to share ideas, learnings, developments related to web accessibility, inclusive design, and other disability support services.
+  </p>
+</section>
+
+
+<section>
+  <h2>We can help</h2>
+  <p>
+    One of our missions is to encourage new speakers and speakers from underrepresented groups to prepare and present their talks!
+    We would be happy to brainstorm your ideas, to review and advise you on how to produce a slide deck. We can also arrange a rehearsal via video chat with you.
```

Comment:
Shouldn't it be our mission instead of our missions?

Reply:
The wording suggests that we have several missions and encouraging new speakers is one of **them**, so it's correct that way 🙂